### ISSUE TITLE
Accessibility: confusion between caption and alternative text leads to badly accessible images

### ISSUE THREAD

**Issue by AmauryCarrade** [zestedesavoir/zmarkdown#497]
At the time of writing, when you place an image alone on its paragraph with alt text, such as :

```md
![Image’s alternative text](https://github.com/zestedesavoir/zmarkdown/assets/1417570/700c7266-12f4-4604-8e5e-3cbed4c4e204)
```

...the image is displayed as a centred figure, _with the alternative text displayed below the image as a caption_. If you want to display a centred image like this _without_ the so-called "caption", you need to place it in a central block, as follows:

```md
-> ![Image’s alternative text](https://github.com/zestedesavoir/zmarkdown/assets/1417570/700c7266-12f4-4604-8e5e-3cbed4c4e204) <-
```

This is annoying, and more importantly, it misleads people as to what an alternative text really is. Alternative texts are not captions, they both have a completely different purpose. A text alternative is a precise and concise description of the image, useful for those who can't see it, whereas a caption is a _complement_ to the image, for example by giving a context, adding sources and credits, etc.

This confusion frequently arises when new zmarkdown users ask why rich text cannot be included in the alt text of images: they think that the alt text is a _caption_. Alternative text is mainly meant to be spoken out loud, written on a Braille display, or read by a robot (search engine, etc.)—writing it as rich text doesn't make sense.

### I suggest that:
- **we remove this auto-caption feature from alt text to avoid this confusion** (the `Figure:` attribute can still be used for a _real_ caption);
- on the user documentation side—but that's not a concern for this repository—we explain the difference.

**Comment by StaloneLab** [comment]
I think we already mentioned this subject somewhere; anyway, I agree on the problem: the way we currently handle alternative texts and captions is misleading.
However, before changing this behavior, we need to think about what to do with the LaTeX, because we currently do *not* render the alternative text in LaTeX anywhere else than in the image caption, as far as I know (when no caption is given, otherwise it is not displayed at all).

Do you have an idea of what would be a good way to integrate this alternative text in LaTeX?
Depending on what is chosen, implementation of ZMarkdown side could be more or less complicated, and we might also decide to update the template.

As a final thought, I shall note that this is a non-backward-compatible change, so might wait a bit before being released, even if changes are ready soon.
